e2e/pages/Helpers/Helpers.ts

```diff
@@ -7,7 +7,7 @@ class Helpers {
     this.page = page;
   }
 
-  async getTodaysDate(): Promise<string> {
+  static async getTodaysDate(): Promise<string> {
```

Comment:
Just out of curiosity what does 'static' change here?

Reply:
[MDN docs on this.](https://dev.azure.com/redcrossnl/121%20Platform/_workitems/edit/28280)

In this example, we aren't using any of the class properties in this function, so there should be no need to make an instance of the class just to be able to use this function. See also where the usage of the function has been changed as a consequence.